src/telemetryReporter.ts

```diff
@@ -79,7 +88,7 @@ export default class TelemetryReporter extends vscode.Disposable {
     // __GDPR__COMMON__ "common.vscodemachineid" : { "endPoint": "MacAddressHash", "classification": "EndUserPseudonymizedInformation", "purpose": "FeatureInsight" }
     // __GDPR__COMMON__ "common.vscodesessionid" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
     // __GDPR__COMMON__ "common.vscodeversion" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
-    private getCommonProperties(): { [key: string]: string } {
+     private getCommonProperties(): { [key: string]: string } {
```

Comment:
Extra whitespace?

Reply:
Accidental formatting

Reply:
Looks like there is still a whitespace change here and below?